packages/design-system/src/utils/functions.ts

```diff
@@ -14,3 +14,24 @@ export const openLink = (link: string, newTab = false) => {
 
 export const isLast = (index: number, array: unknown[]) =>
   index === array.length - 1
+
+const SCROLL_RETRY_TIMEOUT = 500
+export const makeScrollToIndexFailedHandler = (
```

Comment:
I think the function should not be here. It's not global-scoped. Let's move it under `formStepHeader` folder in a filed named `utils.ts`. Also, I'd like to have this function unit tested (it's a valuable logic in my opinion :))